_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_vaapi.cpp

```diff
@@ -792,23 +792,12 @@ mfxStatus VAAPIEncoder::CreateAuxilliaryDevice(
 
     if (p.CodeName >= MFX_PLATFORM_SKYLAKE)
```

Comment:
Perhaps, just 
m_caps.MaxEncodedBitDepth = p.CodeName >= MFX_PLATFORM_KABYLAKE ? 1 : 0;

Reply:
agree

Reply:
I think it is not correct because m_caps.MaxEncodedBitDepth may be 0,1,2,3

Reply:
in your implementation it accepts only two values:
    if (p.CodeName >= MFX_PLATFORM_SKYLAKE)
    {
        m_caps.MaxEncodedBitDepth = 0;
    }
    if (p.CodeName >= MFX_PLATFORM_KABYLAKE)
    {
        m_caps.MaxEncodedBitDepth = 1;
    }